src/control/lib/spdk/ctests/nvme_control_ut.c

```diff
@@ -0,0 +1,286 @@
+/**
+* (C) Copyright 2019 Intel Corporation.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*
+* GOVERNMENT LICENSE RIGHTS-OPEN SOURCE SOFTWARE
+* The Government's rights to use, modify, reproduce, release, perform, display,
+* or disclose this software are subject to the terms of the Apache License as
+* provided in Contract No. 8F-30005.
+* Any reproduction of computer software, computer software documentation, or
+* portions thereof marked with this legend must also reproduce the markings.
+*/
+
+#include <stdarg.h>
+#include <stddef.h>
+#include <setjmp.h>
+#include <cmocka.h>
+
+#include <gurt/types.h>
+#include <spdk/stdinc.h>
+#include <spdk/nvme.h>
+#include <spdk/env.h>
+#include "nvme_internal.h"
+
+#include "../include/nvme_control.h"
+#include "../include/nvme_control_common.h"
+
+#if D_HAS_WARNING(4, "-Wframe-larger-than=")
```

Comment:
Is 4 the right number here?  I thought this warning was added later than gcc 4

Reply:
I not sure I'm afraid, just copy and paste from drpc_tests.c, seems to work as intended at least locally

Reply:
I didn't have an easy way to look.  Yes, looks right.